File: js/fibonacci.js

const fibonacci = (num) => {


  // base cases 0 or 1
  if (num === 0) {
    return 0;
  }
  if (num === 1) {
    return 1;
  }
  // recursive call F(n-1) + F(n-2)
  return fibonacci(num - 1) + fibonacci(num - 2)

}

module.exports = {fibonacci}

File: python/fibonacci.py
def fibonacci(n):
    a = 0
    b = 1
    
    if (n == 0):
        return a
    elif (n == 1):
        return b

    f = 0

    while (n > 1):
        f = a + b
        a = b
        b = f
        n -= 1

    return f

